src/components/game/component.jsx

```diff
@@ -1,14 +1,32 @@
 import React from 'react';
-import { Container } from 'react-bootstrap';
+import { Container, Button } from 'react-bootstrap';
 import PropTypes from 'prop-types';
 
 import Board from '../board';
 
-const Game = ({ history, moveNumber }) => {
+const Game = ({ history, moveNumber, onClick, jumpTo }) => {
   const current = history[moveNumber];
+  const moveMap = history.map((eachMove, move) => {
+    const moveHistory = move ? `Go to move #${move}` : 'Restart';
+    return (
+      <Button
+        size="sm"
+        variant="outline-dark"
+        key={[move]}
+        onClick={() => jumpTo(move)}
+      >
+        {moveHistory}
+      </Button>
+    );
+  });
+
   return (
     <Container className="game">
-      <Board squares={current.squares} />
+      <div className="playerMoves">{moveMap}</div>
```

Comment:
The map function can be run here and props assigned as needed

Reply:
Do you mean to create a mapping function for the squares?

Reply:
Could elaborate by props assigned as needed, please?